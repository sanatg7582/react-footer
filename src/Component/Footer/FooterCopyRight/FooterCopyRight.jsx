import React from "react";
import PropTypes from "prop-types";
import styles from "./FooterCopyRight.module.scss";
import Text from "../../Shared/Text";
/**
 * FooterTermsAndCondition function for print label below copyright section
 * @param {string} tnccontent
 */
const FooterTermsAndCondition = tnccontent => {
  return (
    <div className={styles.tnccontent}>
      <Text heading={tnccontent} />
    </div>
  );
};
/**
 * This Fucntion for copyright content
 * @param {string} copyrighClass
 * @param {string} copyrightContent
 * return Element
 */
const FooterCopyRight = props => {
  const { copyrightcontent, tnccontent } = props;
  return (
    <div className={styles.footerCopyrRight}>
      <Text heading={copyrightcontent} />
      {FooterTermsAndCondition(tnccontent)}
    </div>
  );
};
// default props value
FooterCopyRight.defaultProps = {
  copyrightcontent: ""
};

// propType validation
FooterCopyRight.propTypes = {
  copyrightcontent: PropTypes.string
};
export default FooterCopyRight;
