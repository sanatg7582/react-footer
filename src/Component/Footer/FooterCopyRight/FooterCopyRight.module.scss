.footerCopyrRight {
  position: relative;
  color: #fff;
  float: left;
  top: 270px;
  span {
    padding: 0px 600px;
    font-size: 14px;
  }
  .tnccontent {
    padding: 25px;
    text-align: center;
  }
}
