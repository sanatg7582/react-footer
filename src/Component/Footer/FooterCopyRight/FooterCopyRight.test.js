import React from "react";
import { mount } from "enzyme";
import FooterCopyRight from "./FooterCopyRight";
describe("Load FooterCopyRight with out crash", () => {
  const wrapper = mount(<FooterCopyRight />);
  it("render with out crash", () => {
    expect(wrapper).toBeTruthy();
  });
  it("Snapshot test", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
