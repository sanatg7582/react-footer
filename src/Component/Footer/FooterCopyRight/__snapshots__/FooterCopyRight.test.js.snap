// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Load FooterCopyRight with out crash Snapshot test 1`] = `ReactWrapper {}`;
